modules/installation-deployment-manager-security.adoc

```diff
@@ -0,0 +1,153 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_gcp_user_infra/installing-gcp-user-infra.adoc
+
+[id="installation-deployment-manager-security_{context}"]
+= Deployment Manager template for firewall rules and IAM roles
+
+You can use the following Deployment Manager template to deploy the security
+objects that you need for your {product-title} cluster:
+
+.`03_security.py` Deployment Manager template
+[source,yaml]
+----
+def GenerateConfig(context):
+
+    resources = [{
+        'name': context.properties['infra_id'] + '-api',
+        'type': 'compute.v1.firewall',
+        'properties': {
+            'network': context.properties['cluster_network'],
+            'allowed': [{
+                'IPProtocol': 'tcp',
+                'ports': ['6443']
+            }],
+            'sourceRanges':  ['0.0.0.0/0'],
+            'targetTags': [context.properties['infra_id'] + '-master']
+        }
+    }, {
+        'name': context.properties['infra_id'] + '-mcs',
+        'type': 'compute.v1.firewall',
+        'properties': {
+            'network': context.properties['cluster_network'],
+            'allowed': [{
+                'IPProtocol': 'tcp',
+                'ports': ['22623']
+            }],
+            'sourceRanges':  [
+                context.properties['network_cidr'],
+                context.properties['master_nat_ip'],
+                context.properties['worker_nat_ip']
+            ],
+            'targetTags': [context.properties['infra_id'] + '-master']
+        }
+    }, {
+        'name': context.properties['infra_id'] + '-health-checks',
+        'type': 'compute.v1.firewall',
+        'properties': {
+            'network': context.properties['cluster_network'],
+            'allowed': [{
+                'IPProtocol': 'tcp',
+                'ports': ['6080', '22624']
+            }],
+            'sourceRanges':  ['35.191.0.0/16', '130.211.0.0/22'],
```

Comment:
sourceRanges should be: `['35.191.0.0/16', '209.85.152.0/22', '209.85.204.0/22']`
We recently uncovered this in a bug, cf.: 

openshift/installer#2481 - ipi. PR provides more context
openshift/installer#2488 - upi-version